### ISSUE TITLE
Can't "add child" in Chart of Accounts

### ISSUE THREAD

**Issue by VicentGJ** [frappe/erpnext#26697]
## Description of the issue
Clicking in the "Add Child"buttom does nothing and prints an error to the browser terminal
## Context information (for bug reports)

**Output of `bench version`**
```
erpnext 13.x.x-develop
frappe 13.x.x-develop
```

## Steps to reproduce the issue

1. Go to Charts of Accounts in the accounting module
2. Select any of the root folders and click "Add Child"

### Observed result
Nothing happens.
### Expected result
Should display the creation dialog.
### Stacktrace / full error message

```
Uncaught TypeError: me.new_node is not a function
    click account__tree_js:158
    get_toolbar tree.js:299
    jQuery 8
    get_toolbar tree.js:298
    get_toolbar tree.js:288
    make_node_element tree.js:117
    <anonymous> tree.js:81
    add_node tree.js:124
    render_node_children tree.js:176
    each jQuery
    render_node_children tree.js:175
    load_children tree.js:156
    promise callback*frappe.run_serially/< dom.js:265
    run_serially dom.js:263
    load_children tree.js:154
    toggle_node tree.js:214
    expand_node tree.js:204
    setup_root_node tree.js:98
    <anonymous> tree.js:29
    make_tree treeview.js:154
    change treeview.js:116
    set base_control.js:179
    promise callback*frappe.run_serially/< dom.js:265
    run_serially dom.js:263
    set base_control.js:172
    validate_and_set_in_model base_control.js:194
    parse_validate_and_set_in_model base_control.js:158
    change_handler data.js:121
    jQuery 7
    make_filters treeview.js:123
    each jQuery
    make_filters treeview.js:100
    TreeView treeview.js:41
    make treeview.js:21
desk.bundle.P64XIPLD.js line 481 > eval:158:8
```

## Additional information

Arch Linux, `ERPNext` manual installation.

**Comment by ankush** [comment]
Note: Only affects `develop` branch.

**Comment by OmarAhmedAlattas** [comment]
Dear All

I do have the same issue, I have the same versions (erpnext 13.x.x- develop, frappe 13.x.x-develop) installed on premise on both my PC and Laptop and both have the same issue.
Is there any solution for that or do we have to switch to production

**Comment by nextchamp-saqib** [comment]
Resolved By #28413
